app/src/main/java/layout/mapFragment.java

```diff
@@ -323,27 +327,22 @@ public void onMapClick(final LatLng point) {
         mSelectedMarker = null;
     }
 
-    /**
-     * Enables the My Location layer if the fine location permission has been granted.
-     */
-    private void enableMyLocation() {
-        if (ContextCompat.checkSelfPermission(getActivity(), Manifest.permission.ACCESS_FINE_LOCATION)
-                != PackageManager.PERMISSION_GRANTED) {
-            // Permission to access the location is missing.
-            //// TODO: 19-Mar-17 ask for permission here
-            //PermissionUtils.requestPermission(this, LOCATION_PERMISSION_REQUEST_CODE,
-            //        Manifest.permission.ACCESS_FINE_LOCATION, true);
-        } else if (mMap != null) {
-            // Access to the location has been granted to the app.
-            mMap.setMyLocationEnabled(true);
+    public void verifyLocationPermissions(Activity activity) {
+        Log.d("permTag","in verify perm");
+        // Check if we have location permission
+        int permission = ActivityCompat.checkSelfPermission(activity, Manifest.permission.ACCESS_FINE_LOCATION);
```

Comment:
Verify that we have the permissions to access user location. Else we'll prompt the user.